competitions/Climate2030/ansible/Climate2030.yml

```diff
@@ -5,17 +5,17 @@
   become_user: "{{ deployment_user }}"
   roles:
     - mysql
-  
+
```

Comment:
Hunh, this whitespace changed happened in most of the files.  It's not a big deal here, but I'm wondering if there's some editor setting you have turned on that is likely to cause more extensive non-substantive whitespace changes in the future depending on the state of the files edited.

Reply:
We have an .editorconfig that removes trailing whitespace -- the bug here is that the original author did not IMO!

Reply:
Hey now!